src/main.c

```diff
@@ -61,87 +58,58 @@
 #include "upnp_transport.h"
 #include "upnp_connmgr.h"
 
-static gboolean show_version = FALSE;
-static gboolean show_devicedesc = FALSE;
-static gboolean show_connmgr_scpd = FALSE;
-static gboolean show_control_scpd = FALSE;
-static gboolean show_transport_scpd = FALSE;
-static gboolean show_outputs = FALSE;
-static gboolean daemon_mode = FALSE;
+static int show_version = FALSE;
+static int show_devicedesc = FALSE;
+static int show_connmgr_scpd = FALSE;
+static int show_control_scpd = FALSE;
+static int show_transport_scpd = FALSE;
+static int show_outputs = FALSE;
+static int daemon_mode = FALSE;
 
 // IP-address seems strange in libupnp: they actually don't bind to
 // that address, but to INADDR_ANY (miniserver.c in upnp library).
 // Apparently they just use this for the advertisement ? Anyway, 0.0.0.0 would
 // not work.
-static const gchar *ip_address = NULL;
+static const char *ip_address = NULL;
```

Comment:
sorry, this changed to `interface_name` in the meantime (didn't see your pull request until now)